src/parameters/update_parameters.jl

```diff
@@ -571,14 +569,14 @@ function _update_pwl_cost_expression(
     ::Type{T},
     component_name::String,
     time_period::Int,
-    cost_data::Vector{NTuple{2, Float64}},
+    cost_data::PSY.PiecewiseLinearPointData,
 ) where {T <: PSY.Component}
     pwl_var_container = get_variable(container, PieceWiseLinearCostVariable(), T)
     resolution = get_resolution(container)
     dt = Dates.value(Dates.Second(resolution)) / SECONDS_IN_HOUR
     gen_cost = JuMP.AffExpr(0.0)
     slopes = PSY.get_slopes(cost_data)
-    upb = PSY.get_breakpoint_upperbounds(cost_data)
+    upb = PSY.get_x_lengths(cost_data)
```

Comment:
Not sure about the name for this function. @sourabhdalvi what do you think?

Reply:
My issue with just saying breakpoints is that it's specifically the difference between `x[n]` and `x[n-1]`, whereas from my understanding "breakpoints" can sometimes refer to just the x values themselves. Is that correct?

Reply:
I agree with you from the data point of view but the variables for PWL representation are called breakpoints and these will return the upper bounds for the break point variables in the optimization problem

Reply:
Done with a `get_breakpoint_upperbounds` alias in `powersystem_utils.jl`